packages/next/package.json

```diff
@@ -7,11 +7,11 @@
   "scripts": {
     "build": "node build.js",
     "build-dev": "node build.js --dev",
-    "test": "jest --env node --verbose --bail --runInBand",
-    "test-unit": "yarn test test/unit/",
-    "test-next-local": "jest --env node --verbose --bail --testTimeout=360000 test/integration/*.test.js test/integration/*.test.ts",
-    "test-next-local:middleware": "jest --env node --verbose --bail --useStderr --testTimeout=360000 test/integration/middleware.test.ts",
```

Comment:
Why was `useStderr` removed?

Reply:
these tests get passed `-- --listTests` by the chunk script and thus can't have any other arguments already in there. they seem to still run and pass fine without it so we went with it